lib/matplotlib/mathtext.py

```diff
@@ -464,8 +454,9 @@ def set_canvas_size(self, w, h, d):
         Set the size of the buffer used to render the math expression.
         Only really necessary for the bitmap backends.
         """
-        self.width, self.height, self.depth = ceil(w), ceil(h), ceil(d)
-        self.mathtext_backend.set_canvas_size(self.width, self.height, self.depth)
+        self.width, self.height, self.depth = map(np.ceil, [w, h, d])
+        self.mathtext_backend.set_canvas_size(
```

Comment:
Was this too long or something?

Reply:
yes

Reply:
Was there a pep8ignored defined that can be removed now?

Reply:
there's a bunch of other long lines, let's keep that for another pr.